packages/hotdog/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "hotdog"
+version = "0.1.0"
+edition = "2018"
+publish = false
+build = "build.rs"
+
+[lib]
+path = "pkg.rs"
+
+[[package.metadata.build-package.external-files]]
+# TODO: fix
+url = "http://local.fix.this/hotdog-main.tar.gz"
```

Comment:
https://github.com/bottlerocket-os/hotdog

We've got public sources now :smile: